generic/utils/ras.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+import os
+import commands
+from avocado import Test
+from avocado import main
+from avocado.utils import process
+from avocado.utils.software_manager import SoftwareManager
+
+
+class RASTools(Test):
+
```

Comment:
Would you please add docstring explaining the purpose of this? (basically what you described in the first comment of this PR + you can mention the purpose of some of the scripts to help people understand what is the purpose of this file).